paddlenlp/transformers/model_utils.py

```diff
@@ -203,19 +203,19 @@ def from_pretrained(cls, pretrained_model_name_or_path, *args, **kwargs):
                 # Load from local directory path
                 model = BertForSequenceClassification.from_pretrained('./my_bert/')
         """
-        pretrained_models = list(cls.pretrained_init_configuration.keys())
         resource_files = {}
         init_configuration = {}
         load_state_as_np = kwargs.pop("load_state_as_np", False)
 
         # From built-in pretrained models
-        if pretrained_model_name_or_path in pretrained_models:
+        if pretrained_model_name_or_path in cls.pretrained_init_configuration:
```

Comment:
there are some grammar sugar in python:

```python
'key' in dict_data
```

is equal is to :

```python
'key' in list(dict_data.keys())
```

And there are more improvements:

* time complexity: O(n) -> O(1)
* memory complexity: O(n) -> O(0)

So I change this code.